src/main/java/org/kainos/ea/validator/UpdateJobRoleValidator.java

```diff
@@ -0,0 +1,31 @@
+package org.kainos.ea.validator;
+
+import org.kainos.ea.cli.UpdateJobRoleRequest;
+
+public class UpdateJobRoleValidator {
+    public boolean validate(UpdateJobRoleRequest jobRole)
+    {
+        if (jobRole == null) {
+            return false;
+        }
+        if (jobRole.getJobRoleName() == null || jobRole.getJobRoleName().length() > 64) {
+            return false;
+        }
+        if (jobRole.getJobSpecSummary() == null) {
+            return false;
+        }
+        if (jobRole.getBand() == null || jobRole.getBand().getBandID() > 32767) {
```

Comment:
For band and capability you probably want to check the value exists in the database as well as checking it's a number, so you return a 400 not a 500 back to the end user